tests/queries/0_stateless/02346_text_index_function_hasAnyAllTokens_partially_indexed.reference

```diff
@@ -0,0 +1,72 @@
+-- { echoOn }
+SELECT * FROM explain_indexes;
+Expression ((Project names + Projection))
+  Aggregating
+    Expression (Before GROUP BY)
+      Expression ((WHERE + Change column names to column identifiers))
```

Comment:
There is a chance that one of these `Expression`s are different on some CI builds - I'm not totally sure if setting `parallel_replicas_local_plan` helps, however I'd suggest to drop unnecessary fields from the explain output and only collect the index info with something like that:
```
CREATE VIEW explain_indexes AS (
SELECT trimLeft(explain) AS explain FROM (
        EXPLAIN indexes = 1
        SELECT groupArray(id) FROM tab WHERE hasAnyTokens(message, ['def'])
    )
    WHERE explain LIKE '%Description:%' OR explain LIKE '%Parts:%' OR explain LIKE '%Granules:%'
    LIMIT 2, 3
);
```